.travis.yml

```diff
@@ -26,7 +26,7 @@ deploy:
   on:
     tags: true
     branch: master
-    condition: "$TRAVIS_PYTHON_VERSION == 3.6"
+    skip_existing: true
```

Comment:
@wizmer has pointed out supposedly more proper way to limit PyPI upload to one environment